apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/MetricsConfig.java

```diff
@@ -60,13 +60,22 @@ interface TenantRequestCounterConfig {
     @WithDefault("error")
     String errorTag();
 
-    /** @return The tag key for user-agent flag, defaults to <code>error</code>. */
+    /** @return The tag key for user-agent flag, defaults to <code>user_agent</code>. */
     @NotBlank
     @WithDefault("user_agent")
```

Comment:
this should be `user.agent` and micrometer would do the conversion.. we should stick to the dot convention..

Reply:
I'm not going to do this. It's working currently and I'm not confident of finding all places I would need to change to handle this consistently.